examples/common/scripts/vtorc-up.sh

```diff
@@ -11,7 +11,7 @@ vtorc \
   $TOPOLOGY_FLAGS \
   --logtostderr \
   --alsologtostderr \
-  --config="${script_dir}/../vtorc/config.json" \
+  --config-path="${script_dir}/../vtorc/config.json" \
```

Comment:
According to the docs, we need to provide both `--config-path` and either `--config-file` or `--config-name`, though it seems that providing just the file name as path works.
Let's use two flags in the example so that people looking at it can see the canonical usage. Also, it will be nice to provide a different format than JSON for the local example, since e2e tests also use JSON.